internal/cephfs/core/clone.go

```diff
@@ -54,6 +55,14 @@ func (cs cephFSCloneState) ToError() error {
 	return nil
 }
 
+func (cs cephFSCloneState) GetProgressReport() admin.CloneProgressReport {
+	return admin.CloneProgressReport{
```

Comment:
cant we return `cs.progressReport` in other works do we need this helper function?

Reply:
I initially opted for that, but since the struct is not exposed to functions outside the `core` pkg, we can't directly access the members of the struct.
Hence, a helper function.